modules/ingest-common/src/internalClusterTest/java/org/elasticsearch/ingest/common/IngestRestartIT.java

```diff
@@ -326,4 +336,78 @@ public boolean validateClusterForming() {
         source = client().prepareGet("index", "fails").get(timeout).getSource();
         assertNull(source);
     }
+
+    /**
+     * This test is for confirming that forwarded bulk requests do not use system_write thread pool
+     * for non-system indexes. Before this fix, we were using system_write thread pool for all forwarded
+     * bulk requests causing the system_write thread pool to get overloaded.
+     */
+    public void testForwardBulkWithSystemWritePoolDisabled() throws Exception {
+        // Create a node with master only role and a node with ingest role
+        final String masterOnlyNode = internalCluster().startMasterOnlyNode();
+        final String ingestNode = internalCluster().startNode();  // What roles does a default node have ?
```

Comment:
In stateful, by default a node has all roles. 
* https://www.elastic.co/guide/en/elasticsearch/reference/current/modules-node.html#node-roles
* https://github.com/elastic/elasticsearch/blob/main/server/src/main/java/org/elasticsearch/node/NodeRoleSettings.java#L19-L27

Please remove the comment in the code. I suggest we do _not_ use code level comment for personal questions. Instead you can use GitHub comment for any questions which will be addressed as part of the review process. Thanks!

Reply:
Got it. Done.